src/components/BookCard.js

```diff
@@ -17,7 +17,7 @@ function BookCard({ book }) {
         <div className="options">
           <span className="comments">Comments</span>
           {' '}
-          |
+          ||
           <button
             type="button"
             className="remove"
```

Comment:
- [ ] As said above let's consider deleting one separated bar between the Comments, Remove button, and Edit since you do have two 😜